custom/covid/tasks.py

```diff
@@ -17,6 +18,8 @@
 from dimagi.utils.logging import notify_exception
 from django.conf import settings
 
+RATE_LIMIT = getattr(settings, 'USH_PRIME_RESTORE_RATE_LIMIT', 100)
```

Comment:
Is this interpreted as tasks per second? So this rate limit tells celery to pick up at most 100 of these tasks per second? That seems pretty high, I'd estimate it's high enough to saturate the queue's processors for as long as it takes to complete all of them

Reply:
That's correct. I did ask about this on slack a while ago but I didn't get a response: https://dimagi.slack.com/archives/C0B44C0Q2/p1613382246276800?thread_ts=1613382157.276100&cid=C0B44C0Q2. The traffic on the `async_restore_queue` is very low (roughly 25 per hour during the time when these tasks are run) so the is quite low.

Either way, I have set up a new queue for these and other USH tasks so that they won't impact any other queues: https://github.com/dimagi/commcare-cloud/pull/4534, https://github.com/dimagi/commcare-hq/pull/29181

Reply:
Great thanks. No issues, I just wanted to make sure I was reading it right. Separate queue sounds good, I'll take a look at that PR